.storybook/preview.js

```diff
@@ -48,7 +49,9 @@ addParameters({
 // Storybook Decorators
 addDecorator(withKnobs) // add withKnobs
 addDecorator((story) => (
-  <Provider theme={defaultTheme} colorScheme='light' scale='medium'>
-    {story()}
+  <Provider>
+    <RSProvider theme={defaultTheme} colorScheme='light' scale='medium'>
+      {story()}
+    </RSProvider>
```

Comment:
As an aside should we just refactor the Provider class in ui components to be something like to wrap the react-spectrum provider so we only have to include one Provider?